void tomo_db0::IoLoop::run_loop<true, false, false, false, false>()
--- PRE

+++ POST

@@ -621,7 +621,7 @@

 0b6d [ 3] nopl (%rax)
 0b70 [ 5] mov $0x1,%esi
 0b75 [ 3] mov %rbx,%rdi
-0b78 [ 5] call <tomo_db0::IoLoop::arm_accept(tomo_db0::UrKind)>
+0b78 [ 5] call <tomo_db0::IoLoop::fliplisten_retry_accept(tomo_db0::UrKind)>
 0b7d [ 5] jmp <+0x30b>
 0b82 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0b8d [ 3] nopl (%rax)
